Running-Mastodon/Production-guide.md

```diff
@@ -111,7 +111,12 @@ server {
 
 ## Running in production without Docker
 
-It is recommended to create a special user for mastodon on the server (you could call the user `mastodon`), though remember to disable outside login for it. You should only be able to get into that user through `sudo -u mastodon`.
+It is recommended to create a special user for mastodon on the server (you could call the user `mastodon`), though remember to disable outside login for it. You should only be able to get into that user through `sudo su mastodon`.
```

Comment:
I have my reasons for `sudo su`, if I am wrong in this reasoning, but path seems correct and home is set correctly with `sudo su`

images showing env output differences.
### sudo -u mastodon -s
![screen shot 2017-06-17 at 12 45 31 pm](https://user-images.githubusercontent.com/495512/27249537-59e6418c-535b-11e7-9b58-0fcefc9ad718.png)
### sudo su mastodon
![screen shot 2017-06-17 at 12 46 29 pm](https://user-images.githubusercontent.com/495512/27249538-62b88388-535b-11e7-91a0-a6116bbda86f.png)

I will gladly accept being wrong, but from these outcomes I believe having the correct home is a good idea as mastodon user parts of the tutorial refer to `~/.rbenv` and etc